src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -190,6 +196,16 @@ class BeamSkimmer @Inject()(val beamConfig: BeamConfig, val beamServices: BeamSe
     }
   }
 
+  def getPreviousSkimValueOrDefault(time: Int, mode: BeamMode, orig: Id[TAZ], dest: Id[TAZ]): Skim = {
+    previousSkims.get((timeToBin(time), mode, orig, dest)) match {
+      case someSkim @ Some(_) => someSkim.get.toSkimExternal
```

Comment:
I think it can be rewritten like `case Some(skim) => skim.toSkimExternal`

Reply:
ok